### PR TITLE
feat(azure-theme): bootstrap new package

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
Per documentation, initial PR introduces scaffolding. 
Logic to come in next PR.

Original PR: https://github.com/microsoft/fluentui-contrib/pull/177

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -12,6 +12,7 @@ packages/react-headless-provider/ @microsoft/teams-prg
 packages/react-resize-handler/ @microsoft/teams-prg
 packages/stylelint-plugin/ @microsoft/teams-prg
 packages/react-themeless-provider/ @spmonahan
+packages/azure-theme/ @microsoft/azure-ux-design-engineering
```
Comment (Jacqueline-ms) [inline .github/CODEOWNERS:15]: Can we grant writing access to my new team @microsoft/azure-ux-design-engineering ?

Comment (Hotell) [reply .github/CODEOWNERS:15]: done

**Comment by Hotell** [comment]
as part of boostraping I'd suggest to include storybook as well  `yarn nx generate @fluentui-contrib/nx-plugin:configure-storybook --name=azure-theme`

**Comment by Jacqueline-ms** [comment]
Thanks @Hotell . I included the component commands `yarn nx generate component <name> <componentName>` for our 3 subthemes: azure light, azure dark, azure high contrast.

**Review by Hotell — APPROVED** [review]

**Pull request merged**